app/presenters/tree_builder_vms_and_templates.rb

```diff
@@ -54,7 +54,7 @@ def reparent_hidden_folders(tree)
   def prune_rbac(tree)
     vms = extract_vms(tree).uniq
 
-    allowed_vm_ids, _attrs = Rbac.search(:targets => vms, :results_format => :ids)
+    allowed_vm_ids = Rbac.filtered(vms, :results_format => :ids)
```

Comment:
`filtered` doesn't support different results formats (and I strongly feel it should not)